channels/chan_usrp.c

```diff
@@ -371,9 +371,11 @@ static int usrp_indicate(struct ast_channel *ast, int cond, const void *data, si
 	switch (cond) {
 	case AST_CONTROL_RADIO_KEY:
 		p->txkey = 1;
+		ast_debug(1, "Channel %s: ACRK TX ON.\n", ast_channel_name(ast));
```

Comment:
What is ACRK ?

Reply:
The only part of that I understand is the K.

It ment something to someone else.

Reply:
My first thought when I noticed these two messages was that one indicated keying (K, TX ON) and the other un-keying (UK, TX OFF).  Looking more carefully at the other drivers and the messages here are not unique to usrp.